sycl/include/CL/sycl/queue.hpp

```diff
@@ -61,6 +63,8 @@
 __SYCL_INLINE_NAMESPACE(cl) {
 namespace sycl {
 
+class AssertInfoCopier;
```

Comment:
```suggestion
class __AssertInfoCopier;
```
?

Reply:
Another suggestion is to move to `detail` namespace

Reply:
Shifted to `detail` namespace